lib/sdk.dart

```diff
@@ -335,8 +324,10 @@ class SdkLocationJob extends Job {
 
   SdkLocationJob(this.sdkManager) : super('Auto locate SDK');
 
+  bool get quiet => true;
```

Comment:
Yup, it's an override from the parent. It indicates that a 'success' toast should not be shown after the job finishes.